sonarqube/content.md

```diff
@@ -30,68 +45,119 @@ $ mvn sonar:sonar -Dsonar.host.url=http://$(boot2docker ip):9000
+### Database
 
-For example, on Linux, you can set the recommended values for the current session by running the following commands as root on the host:
+By default, the image will use an embedded H2 database that is not suited for production.
 
-```console
-sysctl -w vm.max_map_count=262144
-sysctl -w fs.file-max=65536
-ulimit -n 65536
-ulimit -u 4096
-```
+> Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
 
-## Advanced configuration
+Setup a database by following the "Installing the Database" section of https://docs.sonarqube.org/latest/setup/install-server/.
 
-### Database configuration
+### Use bind-mounted folders
 
-By default, the image will use an embedded H2 database that is not suited for production.
+The images contain the SonarQube installation at `/opt/sonarqube`. You need to use bind-mounted folders to override selected files or directories :
 
-> Warning: Only a single instance of SonarQube can connect to a database schema. If you're using a Docker Swarm or Kubernetes, make sure that multiple SonarQube instances are never running on the same database schema simultaneously. This will cause SonarQube to behave unpredictably and data will be corrupted. There is no safeguard until [SONAR-10362](https://jira.sonarsource.com/browse/SONAR-10362).
+-	`/opt/sonarqube/conf`: configuration files, such as `sonar.properties`
+-	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
+-	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
+-	`/opt/sonarqube/extensions`: plugins, such as language analyzers
 
-### Option 1: Use parameters via Docker environment variables
+### First installation
 
-The production database is configured with the following SonarQube properties used as environment variables: `sonar.jdbc.username`, `sonar.jdbc.password` and `sonar.jdbc.url`.
+Follow these steps on your first installation:
 
+1. Create a `sonarqube_home` folder and create a var env `$SQ_HOME` pointing to it:
+```console
+$ mkdir /path/to/your/filesystem/sq_home
+$ export SQ_HOME=/path/to/your/filesystem/sq_home
+```
+2. Initialize SQ_HOME folder tree with `--init`. This will initialize the default configuration, copy embedded plugins, and prepare the data folder:
+```console
+$ docker run --rm \
+ -v $SQ_HOME/conf:/opt/sonarqube/conf \
+ -v $SQ_HOME/extensions:/opt/sonarqube/extensions \
+ -v $SQ_HOME/data:/opt/sonarqube/data \
+ sonarqube --init
+```
+3. Configure sonar.properties & follow the normal first installation procedure (TODO add a link).
```

Comment:
I'm not sure exactly which documentation we are looking for here.

Reply:
well, i don't know either. Something explaining what should be looked at in the sonar.properties for example. The same steps you would follow for a classical unzip installation